dotCMS/src/main/java/com/dotcms/api/system/event/local/type/SingleEndpointFailureEvent.java

```diff
@@ -0,0 +1,4 @@
+package com.dotcms.api.system.event.local.type;
+
+public class SingleEndpointFailureEvent {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2094)